### DISCUSSION TITLE
Windows example and vcpkg?

### DISCUSSION THREAD

**Discussion by wuqi** [Tencent/tgfx#38] [General] [answered]
It seems that there are no examples for the Windows platform, and I hope to be able to install directly using vcpkg or xrepo.

**Comment by domchen** [comment, marked as answer]
We are currently developing the Windows demo, which will be available soon. Similar to Skia, TGFX offers various build options. However, it might not be feasible to provide a universal prebuilt library of TGFX that caters to all user cases. For this reason, we recommend utilizing the TGFX library directly from the source code.

**Comment by domchen** [comment]
The Windows example has been merged into the main branch through Pull Request #76 .
